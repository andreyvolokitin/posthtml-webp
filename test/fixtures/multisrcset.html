<!doctype html>
<html>
    <body>
        <img srcset="photo-lg.jpg 1000w, photo-md.jpg 500w, photo-sm.jpg 250w, photo.jpg" src="photo.png">
    </body>
</html>
