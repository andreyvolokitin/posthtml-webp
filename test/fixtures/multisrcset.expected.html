<!doctype html>
<html>
    <body>
        <picture><source type="image/webp" srcset="photo-lg.webp 1000w, photo-md.webp 500w, photo-sm.webp 250w, photo.webp"><img srcset="photo-lg.jpg 1000w, photo-md.jpg 500w, photo-sm.jpg 250w, photo.jpg" src="photo.png"></picture>
    </body>
</html>
